components/extractors/other/telemetry-data/telemetry-data.md

```diff
@@ -0,0 +1,292 @@
+| `sharing_type` | Type of the bucket sharing (Data Catalog). <br> Possible values: `none` – bucket not shared, <br> `target` – bucket linked from another project, <br>`source` / `org private` – bucket shared from this project, linkable only by organization members <br> `source` / `org public` – bucket shared from this project, linkable by any project members | `target` |
+| `shared_from_project_id` | Identifier of the source project if the bucket is linked (sharing_type = 'target') | `860_eu-central-1_aws` |
+| `shared_from_bucket` | Identifier of the source bucket if the bucket is linked (sharing_type = 'target') | `in.c-keboola-ex-instagram-152387726` |
+
+
+### kbc_component_configuration
+This table lists the configurations of the components (e.g., a configuration of the AWS S3 extractor).
+
+| **Column** | **Description** | **Example** |  
+| `kbc_component_configuration_id` (PK) | Component configuration identifier | `580_us-east-1_aws_keboola.python- transformation-v2-610931033` |
+| `kbc_component_configuration_url` | URL of the configuration in the KBC project | `https://connection.keboola.com/ admin/projects/580/transformations/ bucket/152336525` |
+| `kbc_project_id` | Foreign key to the KBC project | `580_us-east-1_aws` |
+| `kbc_component_id` | Identifier of the component | `keboola.python-transformation-v2` |
+| `kbc_component` | Name of the component | `Generic` |
+| `configuration_id_num` | Numeric identifier of the configuration | `610931033` |
+| `kbc_component_configuration` | Name of the configuration | `Sample data from Dynamics` |
+| `kbc_configuration_version` | Current version of the configuration | `5` |
+| `kbc_configuration_is_deleted` | Flag determining if the configuration is deleted (`true`, `false`) | `false` |
+| `token_id` | Identifier of the token that created this version of the configuration | `241247` |
+| `token_name` | Name of the token that created this version of the configuration | `martin.matejka@keboola.com` |
+
+
+### kbc_component_configuration_version
+This table shows the version history of the component configuration.
+
+| **Column** | **Description** | **Example** | 
+| `kbc_component_configuration_id` (PK) | Component configuration identifier | `6610_us-east-1_aws_orchestrator-583757303` |
+| `configuration_updated_at` | Datetime of the configuration update | `2020-03-28 20:29:25` |
+| `change_description` | Description of what was changed in the configuration | `Update orchestration notifications` |
+| `configuration_version` (PK) | Version of the configuration | `3` |
+| `last_version` | Flag determining if this is the last version of the configuration (`true`, `false`) | `false` |
+| `token_id` | Identifier of the token that created this version of the configuration | `241247` |
+| `token_name` | Name of the token that created this version of the configuration | `martin.matejka@keboola.com` |
+
+
+### kbc_data_science_sandbox
+This table lists Python/R sandboxes and their consumption.
```

Comment:
link na https://help.keboola.com/transformations/sandbox/ nebo https://help.keboola.com/transformations/workspace/ - podle toho, co to doopravdy je, nebo je tam obojí?

Reply:
Jsou tam jen R/Python workspaces/sandboxes. Ta data jsou strukturou stejná pro AWS i Azure.